en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+        "ContainerMgtInfo": [
+        {
+          "ClusterName": "",
+          "DisplayName": "",
+          "Properties": {
+            "MasterURL": "",
+            "AccessURL": "",
+            "SAToken": "",
+            "Namespace": "default",
+            "Replicas": 1,
+            "BasicSecurityCustomResourceName": "",
+            "OauthSecurityCustomResourceName": "",
+            "JWTSecurityCustomResourceName": ""
+            }
+          }
+        ]
+      }
+    ]
+    ```
+    
+    ``` json tab="Example"
+    "ContainerMgt": [
+      {
+        "Type": "Kubernetes",
+        "ContainerMgtInfo": [
+         {
+           "ClusterName": "docker-desktop",
+           "DisplayName": "cluster-1",
+           "Properties": {
+           "MasterURL": "https://kubernetes.docker.internal:6443",
+           "AccessURL": "mgw.ingress.wso2.com",
+           "SAToken": "eyJhbGciOiJSUzI1NiIsImtpZCI6Ijg2UlYySEFkaVBOVG4zZ3FhWERXQUxQVUQ0bjlzOXE3dFhpbUZNZlFiRjQifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJ3c28yIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6IndzbzJhbS1wYXR0ZXJuLTEtc3ZjLWFjY291bnQtdG9rZW4tZGR0OTkiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoid3NvMmFtLXBhdHRlcm4tMS1zdmMtYWNjb3VudCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6Ijc3NmRmMjFkLTA3NjItNDM2Zi05ZDIwLTZlYzFkODMxYzc1NSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDp3c28yOndzbzJhbS1wYXR0ZXJuLTEtc3ZjLWFjY291bnQifQ.YwlSgnSnwS7BNP0q0iDHsjlq_r3RhobG-SrduuEi35VXNfipPsK3UVJcCBmDKQOzKAxRG9415h2pr3cS-QsM6PR_UU2UWlXiUO-3UtSMqVY48Ek_1wdfmfCkiE7IOeua_58CV15tXtMLom0Oh27nkmWGfTHVvImQnqho4nv26BKZH4vKWgkw0HpMt73KHdN6SlVMK0cynktO5H-2A4V2rh-uL-OUXCLON_sBypVoPc9PAMxHo-bUe0SqBPiM0SiALu-0-J6dBcwbzPb5g-yUZmHmtuw3O32C304Hgfr-4Dui3X5DSKBSeqlrjjjfrvMVNug0J4JKk3bJ56h0OgujFw",
+           "Namespace": "default",
+           "Replicas": 1,
+           "BasicSecurityCustomResourceName": "",
+           "OauthSecurityCustomResourceName": "",
+           "JWTSecurityCustomResourceName": ""
+            }
+          }
+        ]
+      }
+    ]
+    ```
+ 
+     <html><div class="admonition note">
+        <p class="admonition-title">Note</p>
+        <p> Super tenant users can customize the container management implementations if required. Then customize classes should be added to the `<API-M_HOME>/repository/conf/deployment.toml` file as follows. </p>   
+        
+    ``` json tab="Format"
+    [ContainerMgtConfig]
+    <Environment-type> = "<class-name>"
+    ```  
+   
+    ``` json tab="Example"
+    [ContainerMgtConfig]
+    Kubernetes = "org.wso2.carbon.apimgt.impl.containermgt.K8sManager_CustomClass"
+    ```
+  
+      </div>
+      </html>
+         
+6. Start the server and create an API in publisher
+    
+    After setting up configurations go to the <API-M-HOME>/bin directory and start the server. Then Sign in to the WSO2 API Publisher and create
```

Comment:
unnecessary spaces before "a new API"

Reply:
Shall we point to the document on how to create an API?

Reply:
Done